src/apps/opening-hours-editor/helper.ts

```diff
@@ -0,0 +1,112 @@
+import dayjs from "dayjs";
+import { EventInput } from "@fullcalendar/core";
+import { EventImpl } from "@fullcalendar/core/internal";
+import { DplOpeningHoursListGET200Item } from "../../core/dpl-cms/model";
+
+const formatDateTimeString = (date: string, time: string): string => {
+  return `${date}T${time}:00`;
+};
+
+export const formatCmsEventsToFullCalendar = (
+  data: DplOpeningHoursListGET200Item[]
+): EventInput[] => {
+  return data.map(({ category, date, start_time, end_time, id }) => {
+    return {
+      id: id.toString(),
+      title: category.title,
+      start: formatDateTimeString(date, start_time),
+      end: formatDateTimeString(date, end_time),
+      color: category.color
+    };
+  });
+};
+
+export const formatFullCalendarEventToCmsEvent = (
+  event: EventInput | EventImpl
+): DplOpeningHoursListGET200Item => {
+  const isEventInput = "color" in event; // Check if it's EventInput type
+  const color = isEventInput ? event.color : event.backgroundColor;
+
+  if (!event.title || !color) {
+    throw new Error("Invalid event format");
+  }
+
+  const startDate = dayjs(event.startStr);
+  const endDate = dayjs(event.endStr);
+
+  return {
+    id: Number(event.id),
+    category: {
+      title: event.title,
+      color
+    },
+    date: startDate.format("YYYY-MM-DD"),
+    start_time: startDate.format("HH:mm"),
+    end_time: endDate.format("HH:mm"),
+    branch_id: 0
+  };
+};
+
+export const formatDateStr = (date: Date) => {
+  return dayjs(date).format("YYYY-MM-DDTHH:mm:ssZ");
```

Comment:
This is ISO8601 [as supported by dayjs](https://day.js.org/docs/en/display/as-iso-string) and required by the API. I would prefer that we used this.